js/README.md

```diff
@@ -90,6 +90,19 @@ pnpm ts:check
 
 Currently the following checks are run in CI:
 
+### static checks
+
 - Format check (`pnpm format:check`)
+- Build css (`pnpm css`)
+- Build client (`pnpm build`)
```

Comment:
Is the build client step needed here @pngwn ?

Reply:
Nah it happens as part of the build command I think. Will doublecheck.

Reply:
Yeah it does.

Reply:
Word - thanks for checking!